crates/superposition_types/src/database/models/cac.rs

```diff
@@ -79,6 +79,9 @@ pub struct Dimension {
     pub position: Position,
     pub description: String,
     pub change_reason: String,
+    pub dependency_graph: Option<Value>,
```

Comment:
same for this as well, this also can we have it as 
```suggestion
    pub dependency_graph: Map<String, Value>,
```
and backfill with `{}` ?